lib/index.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
-const Assert = require('@hapi/hoek/lib/assert');
-const Clone = require('@hapi/hoek/lib/clone');
+const Assert = require('@hapi/hoek/assert');
```

Comment:
@Marsup this is the actual line that gives us the error in runtime, sorry for the confusion